templates/patient/card/insurance.html.twig

```diff
@@ -4,7 +4,9 @@
 <ul class="nav nav-tabs mb-2">
 {% for t in ins %}
     <li class="nav-item" role="presentation">
-        <a href="#{{ t.type|attr }}" id="{{ t.type|attr }}-tab" data-toggle="tab" role="tab" aria-controls="{{ t.type|attr }}" aria-selected="{{ loop.first ? "true" : "false" }}" class="nav-link {{ loop.first ? "active" }}">{{ t.type|capitalize|text }}</a>
+        <a href="#{{ t.type|attr }}-{{ t.id|attr }}" id="{{ t.type|attr }}-{{ t.id|attr }}-tab" data-toggle="tab" role="tab" aria-controls="{{ t.type|attr }}-{{ t.id|attr }}" aria-selected="{{ loop.first ? "true" : "false" }}" class="nav-link {{ loop.first ? "active" }}">
```

Comment:
hi @robertdown , added a suffix with `-id` for the `href`, `id` and `aria-controls` here and to the below `tab-content` so it would create a unique match for the different array of insurances